packages/gatsby-plugin-offline/README.md

```diff
@@ -79,3 +79,15 @@ to safely remove the service worker:
 
 This will ensure that the worker is properly unregistered, instead of leaving an
 outdated version registered in users' browsers.
+
+## Gotchas
```

Comment:
```suggestion
 ## Notes
```

Yeah - you're right! I think gotchas has a (deservedly) negative connotation, when in actuality--it's probably better to consider this a brief note!